### PR TITLE
ReaperX Presence Update

### PR DESCRIPTION
**Pull request by InwTanRew**
- Change Color
- Renew Description
- Change user Id
- Delete /backend in presence.ts

### CODE REVIEW

**Review by Bas950 — DISMISSED** [review]

**Comment by InwTanRew** [comment]
Add /backend Again Already Fix backend didn't detected in presence

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of the changes working.

**Comment by InwTanRew** [comment]
9119C59E-7B61-4955-B9A6-394B5127ECDF

here you are

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Comment by Alanexei** [comment]
@InwTanRew It seems that your presence has been modified since you've made the pull request. You will need to resolve those conflicting files.

**Comment by InwTanRew** [comment]
okay wait a second

**Comment by InwTanRew** [comment]
How its conflicting files am confused

**Comment by InwTanRew** [comment]
Updated Logo

File: websites/R/ReaperX/dist/metadata.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+      "id": "237747645790027780"
+    },
+    "service": "ReaperX",
+    "description": {
+      "en": "ReaperX is a Robux Shop (R$) with our professional service, our service is the most popular and lowest priced",
+      "th": "ReaperX ร้านรับเติม ROBUX (R$, โรบัค) ด้วยการบริการเเบบมืออาชีพ บริการของเราเป็นที่นิยมเเละราคาถูก",
+    },
+    "url": "reaperx.in.th",
+    "version": "1.0.1",
+    "logo": "https://i.imgur.com/vbdVmLg_d.jpg",
```
Comment (Bas950) [inline websites/R/ReaperX/dist/metadata.json:13]: logo should at least be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/R/ReaperX/dist/metadata.json (hunk 1 above)
Comment (InwTanRew) [reply websites/R/ReaperX/dist/metadata.json]: Fixed

**Review by Bas950 — APPROVED** [review]

File: websites/R/ReaperX/dist/metadata.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+      "name": "Mxgic",
+      "id": "237747645790027780"
+    },
+    "service": "ReaperX",
+    "description": {
+      "en": "ReaperX is a Robux Shop (R$) with our professional service, our service is the most popular and lowest priced",
+      "th": "ReaperX ร้านรับเติม ROBUX (R$, โรบัค) ด้วยการบริการเเบบมืออาชีพ บริการของเราเป็นที่นิยมเเละราคาถูก",
+    },
+    "url": "reaperx.in.th",
+    "version": "1.0.1",
```
Comment (doomlerd) [inline websites/R/ReaperX/dist/metadata.json:12]: Increase the version.

**Review by doomlerd — CHANGES_REQUESTED** [review]

File: websites/R/ReaperX/dist/metadata.json (hunk 2 above)
Comment (InwTanRew) [reply websites/R/ReaperX/dist/metadata.json]: Here you are, Fixed

**Review by Bas950 — APPROVED** [review]

**Pull request merged**